projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss

```diff
@@ -17,7 +17,13 @@
 /**
  * WP logo menu
  */
-#wpadminbar #wp-admin-bar-wp-logo>.ab-item .ab-icon:before {
+#wpadminbar #wp-admin-bar-wpcom-logo>.ab-item .ab-icon {
+	width: 15px;
```

Comment:
took style from https://github.com/WordPress/wordpress-develop/blob/5a30482419f1b0bcc713a7fdee3a14afd67a1bca/src/wp-includes/css/admin-bar.css#L493 to resolve the broken CSS layout of the wpcom logo